identity/Makefile

```diff
@@ -1,40 +1,34 @@
 IDEN3_DIR = $(HOME)/iden3
-IDENTITIES_FILE = $(IDEN3_DIR)/identities.json
+ISSUED_CLAIM_PATH = $(IDEN3_DIR)/JohnDoe/private/claims
+RECEIVED_CLAIMS_DIR = $(IDEN3_DIR)/AliceWonder/private/received-claims
 
-JD_ID = $(shell jq -r .JohnDoe $(IDEN3_DIR)/identities.json)
-AW_ID = $(shell jq -r .AliceWonder $(IDEN3_DIR)/identities.json)
+.PHONY: all
+all: success-test
 
-CLAIM_NONCE = 2
-CLAIM_FILENAME = $(CLAIM_NONCE)-$(AW_ID).json
-ISSUED_CLAIM_PATH = $(IDEN3_DIR)/JohnDoe/claims/$(CLAIM_FILENAME)
-RECEIVED_CLAIMS_DIR = $(IDEN3_DIR)/AliceWonder/received-claims
+.PHONY: success-test
+success-test: identities issue-claim respond-to-challenge
 
-all: identities issue-claim receive-claim respond-to-challenge
-
-test: identities
-	@echo IDEN3_DIR: $(IDEN3_DIR)
-	@echo IDENTITIES_FILE: $(IDENTITIES_FILE)
-	@echo JD_ID: $(JD_ID)
-	@echo AW_ID: $(AW_ID)
-	@echo CLAIM_FILENAME: $(CLAIM_FILENAME)
-	@echo ISSUED_CLAIM_PATH: $(ISSUED_CLAIM_PATH)
-	@echo RECEIVED_CLAIMS_DIR: $(RECEIVED_CLAIMS_DIR)
+.PHONY: failure-test
+failure-test: identities issue-claim-fails-challenge respond-to-challenge
```

Comment:
The -test targets require the QR code to have been downloaded already, but that's OK -- they're convenient. Perhaps should comment though.

Reply:
yeah, it would be great if we can automate the generation of QR code. But I feel they might become throw-away code once these libs are wrapped into micro-services